apps/asap-server/test/handlers/users/fetch.test.ts

```diff
@@ -99,6 +99,7 @@ describe('GET /users', () => {
         httpMethod: 'get',
         queryStringParameters: {
           search: 'first last',
+          filter: 'Lead PI',
```

Comment:
Should you test for multiple and none?

Reply:
I'm assuming implementation comes later, this is to ensure a `filter` on `qs` doesn't give `400`.